telegram/ext/_updater.py

```diff
@@ -769,7 +769,7 @@ async def bootstrap_set_webhook() -> bool:
             if drop_pending_updates:
                 _LOGGER.debug("Dropping pending updates from Telegram server")
             await self.bot.set_webhook(
-                url=webhook_url,
+                url=webhook_url if webhook_url is not None else "",
```

Comment:
passing an empty string wouldn't make sense. Since this callback function is used only within an `if webhook_url` cause (see line 813), a `type: ignore` would be enough IMO.